packages/terra-date-picker/translations/de.json

```diff
@@ -1,3 +1,4 @@
 {
-  "Terra.datePicker.today": "Heute"
+  "Terra.datePicker.today": "Heute",
+  "Terra.datePicker.openCalendar": "Open Calendar"
```

Comment:
I'm waiting for the translation for this text and the text in terra-search-field. I'll update once I have the translations.